src/MOI/MOI_wrapper.jl

```diff
@@ -298,6 +282,11 @@ mutable struct Optimizer <: MOI.AbstractOptimizer
         model.affine_constraint_info = Dict{Int, ConstraintInfo}()
         model.sos_constraint_info = Dict{Int, ConstraintInfo}()
 
+        model.callback_table = CallbackTable()
+        model.callback_state = CallbackState[]
+        model.callback_cut_data = CallbackCutData()
+
+        # TODO: separate MOI.empty! from initializer
```

Comment:
why?
you are planning to replicate code?

Reply:
nevermind